tests/datasets/test_groupassignee.py

```diff
@@ -1,5 +1,5 @@
 import pytz
-import simplejson as json
+import rapidjson
```

Comment:
This seems unrelated? I know there was talk about enforcing that as a default earlier, but that doesn't seem to have made it into this change.

Reply:
it is indeed unrelated.
It came from the first attempt: https://github.com/getsentry/snuba/pull/857 where it was needed
when I copied over the change to this file I forgot to remove that change